sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/integration/SanityTest.kt

```diff
@@ -24,7 +24,7 @@ class SanityTest(
     private val wallet: TestWallet,
     private val extfvk: String,
     private val extpub: String,
-    private val birthday: Int
+    private val birthday: Long
```

Comment:
If it doesn't get too messy, it would be ideal to use `BlockHeight` as the type for birthday.

Reply:
No problem. In addition to this change, I also had to check the server proper communication, as it seems we experience some troubles in communication with our Lightwalletd server now.